### ISSUE TITLE
Bump version and publish

### ISSUE THREAD

**Issue by woodyrew** [jergason/recursive-readdir#41]
Hi,

Could you bump the version (patch) and publish to npm? [Comparison to previous version for reference](https://github.com/jergason/recursive-readdir/compare/v2.0.0...HEAD).

The following should do it.
```
npm version patch
git push && git push --tags
npm publish
```

Thanks

**Comment by woodyrew** [comment]
@jergason ping :smile:

**Comment by terbooter** [comment]
+1
```
npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
```

**Comment by jergason** [comment]
Closed by 63f93b42f622f5d0d0ae18a303acf63a4d57fa49
